app/components/footer/talk_to_us_component.html.erb

```diff
@@ -17,14 +17,15 @@
                 <p class="telephone">0800 389 2500</p>
               </div>
             </div>
-            <p>Monday to Friday between <strong>8:30am and 5:30pm</strong></p>
+            <p>Monday to Friday between <strong>8:30am and 5:30pm</strong>
+            Chat will be unavailable from 1pm on 24 December until 29 December. It will also be closed on 3 January. Responses may be slower over the Christmas period.</p>
```

Comment:
```suggestion
            <p>Chat will be unavailable from 1pm on 24 December until 29 December. It will also be closed on 3 January. Responses may be slower over the Christmas period.</p>
```